src/main/java/com/deavensoft/paketomat/exceptions/NoSuchCourierException.java

```diff
@@ -0,0 +1,9 @@
+package com.deavensoft.paketomat.exceptions;
+
+public class NoSuchCourierException extends Exception{
+    private static final String MESS = "There is no courier with id ";
+
+    public NoSuchCourierException(Long id){
+        super(MESS + id);
```

Comment:
JSON response in case of errors should be:
{
    "status": 500,
    "error": "Internal Server Error",
    "message": "Error message!"
}